packages/nimble-components/src/dialog/styles.ts

```diff
@@ -18,11 +24,101 @@ import { themeBehavior } from '../utilities/style/theme';
 export const styles = css`
     ${display('grid')}
 
-    dialog {
+    dialog[open] {
+        display: flex;
+        flex-direction: column;
         background-color: ${applicationBackgroundColor};
-        border: ${borderWidth} solid ${popupBorderColor};
-        box-shadow: 0px 2px 3px ${popupBoxShadowColor};
-        max-width: 50%;
+        border: none;
+        box-shadow: 0px 4px 8px #0000004d;
+        padding: 0px;
+        width: 400px;
+        max-height: 600px;
+    }
+
+    header {
+        min-height: 48px;
+        padding: 24px 24px 0px 24px;
+        flex: none;
+        display: flex;
+        flex-direction: column;
+        gap: ${smallPadding};
+    }
+
+    :host([header-hidden]) header {
+        ${
+            /**
+             * Hide content visually while keeping it screen reader-accessible.
+             * Source: https://webaim.org/techniques/css/invisiblecontent/#techniques
+             * See discussion here: https://github.com/microsoft/fast/issues/5740#issuecomment-1068195035
+             */
+            ''
+        }
+        display: inline-block;
+        height: 1px;
+        width: 1px;
+        position: absolute;
+        margin: -1px;
+        clip: rect(1px, 1px, 1px, 1px);
+        clip-path: inset(50%);
+        overflow: hidden;
+        padding: 0;
+    }
+
+    slot[name='title'] {
+        font: ${titlePlus1Font};
+        color: ${titlePlus1FontColor};
+    }
+
+    slot[name='title']::slotted(*) {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+    }
+
+    slot[name='subtitle'] {
+        font: ${subtitleFont};
+        color: ${subtitleFontColor};
+    }
+
+    section {
+        flex: auto;
+        overflow-y: auto;
+        font: ${bodyFont};
+        color: ${bodyFontColor};
+        display: flex;
```

Comment:
As we are defining that the `display: flex` is something that is user facing (the user needs to be aware that they are flex-items in a flex container). It might be worth having explicit tests for it in the unit tests. I'm 50/50 on the value of that.

Like if someone refactored the dialog styles without solid awareness that the content had to stay flex layout because that is part of the public api would a test like that help prevent a regression? I think it might but interested in ideas on it.

Reply:
Without having actually written the test, I think it would be pretty easy to write a test that asserts the section has `display: flex` set on it. I think that would be enough to cause people to pause before changing the styling. Is that what you had in mind?

Reply:
yep! and the test lgtm